### PR TITLE
app-admin/spectre-meltdown-checker: New package

### PR DESCRIPTION
**Pull request by lucianposton**
Closes: https://bugs.gentoo.org/652550
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: app-misc/spectre-meltdown-checker/spectre-meltdown-checker-0.37.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	SRC_URI="https://github.com/speed47/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Spectre & Meltdown vulnerability/mitigation checker for Linux"
+HOMEPAGE="https://github.com/speed47/spectre-meltdown-checker"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
```
Comment (Zlogene) [inline app-misc/spectre-meltdown-checker/spectre-meltdown-checker-0.37.ebuild]: drop this

File: app-misc/spectre-meltdown-checker/spectre-meltdown-checker-0.37.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+DEPEND=""
```
Comment (Zlogene) [inline app-misc/spectre-meltdown-checker/spectre-meltdown-checker-0.37.ebuild]: and this

File: app-misc/spectre-meltdown-checker/spectre-meltdown-checker-0.37.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+RDEPEND="${DEPEND}"
```
Comment (Zlogene) [inline app-misc/spectre-meltdown-checker/spectre-meltdown-checker-0.37.ebuild]: and this

File: app-misc/spectre-meltdown-checker/spectre-meltdown-checker-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	SRC_URI="https://github.com/speed47/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Spectre & Meltdown vulnerability/mitigation checker for Linux"
+HOMEPAGE="https://github.com/speed47/spectre-meltdown-checker"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
```
Comment (Zlogene) [inline app-misc/spectre-meltdown-checker/spectre-meltdown-checker-9999.ebuild]: drop the same lines as I commented above

**Comment by lucianposton** [comment]
moved to app-admin category

**Comment by candrews** [comment]
Can you please add `<upstream>` info to metadata.xml? See https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/ for more info, and there are plenty of examples in the tree.

```
        <upstream>
                <remote-id type="github">speed47/spectre-meltdown-checker</remote-id>
        </upstream>
```
seems about right to me.

Also, the license https://github.com/speed47/spectre-meltdown-checker/blob/master/LICENSE seems to be `GPL-3+`, not `GPL-3`

**Comment by candrews** [comment]
The commit message shouldn't just be "new package" so I've updated that and pushed: c048710cd14c420bb683812e19d86263fcd9ef72

Thank you!

**Pull request closed without merging**